### ISSUE TITLE
Mark Civic Tech Structure as Completed and change visibility to False

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#3895]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
We want to mark the Civic Tech Structure project as `Complete` and make it not visible on the site
- our website accurately reflects the status (Active, Complete, or On Hold) of our projects, 
- and visitors to our website can know the status of our projects.

### Action Items
In `_projects/`, change the following for Civic Tech Structure from
- [x] Status
  ```
  status: Active
  ```
  to
  ```
  status: Complete
  ```
- [x] Visibility
  ```
  visible: true
  ```
  to
  ```
  visible: false
  ```

### Resources/Instructions
- Directory to find the page in once you have it in your IDE: `_projects/civic-tech-structure.md`
- [hackforla.org, Projects page](https://www.hackforla.org/projects/)
- [hackforla.org, Civic Tech Structure page](https://www.hackforla.org/projects/civic-tech-structure)
- Click on the link to see the source file code: https://github.com/hackforla/website/blob/gh-pages/_projects/civic-tech-structure.md?plain=1
- This issue is part of an epic: #3891

**Comment by RyanGehris** [comment]
Availability: Feb 2nd from 3-9
ETA: End of day Feb 2nd

**Comment by RyanGehris** [comment]
I am waiting on confirmation for the following question:

Should the status be changed to "Complete" or "On Hold"? The overview and first check point state that it should be changed to "Complete", but the third bullet point says it should be placed "On Hold"

Won't changing the visibility to false cause users to not be able to see the project or navigate to the project's page? I think that once visibility is changed to false, the real check would be to see if the project card is removed from the page.

**Comment by MattPereira** [comment]
@ExperimentsInHonesty I removed the last action item you wrote
> - [ ] Ensure that the On Hold status is displayed properly on the [projects page](https://www.hackforla.org/projects/) and [Civic Tech Structure page](https://www.hackforla.org/projects/civic-tech-structure)

because it doesn't make sense given that the first action item is to change `status:` to `Complete` and the second action item is to set `visibility:` to `false` which would make completing the last action item impossible

**Comment by MattPereira** [comment]
@RyanGehris I think you are spot on with your plan to verify `visible : false` by ensuring the Civic Tech Structure project card no longer appears on the [projects](https://www.hackforla.org/projects/) page

**Comment by ExperimentsInHonesty** [comment]
Both items were changed. 

> visible: false
> status: Completed

 Good Job
